### PR TITLE
added function to do col major instead of row

### PR DESCRIPTION
**Pull request by jbrowne6**
This sets the required flag for using col major matrices.  A pull request in the packed forest repo is also required to make this work.

### CODE REVIEW

File: R/fpPredict.R [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+		if(class(forest)!="Rcpp_forestPackingRConversion"){
+			stop("forest is not of correct type.")
+		}
+
+		if(is.null(X)){
+			stop("no observations to predict.")
+		}
+
+		X <- as.matrix(X)
+		if(class(X[1,1])=="integer"){
```
Comment (falkben) [inline R/fpPredict.R]: @MrAE did you have a different/preferred way of doing this in R?

File: R/fpRerF.R [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+		##### Check X and Y inputs
+		################################################
+		if(xor(is.null(X), is.null(Y))){
+			stop("X and Y must be set or both must be blank.")
+		}
+
+		if(!is.null(X)){
+			X <- as.matrix(X)
+			if(class(X[1,1])=="integer"){
```
Comment (falkben) [inline R/fpRerF.R]: also here @MrAE ?

File: R/fpPredict.R (hunk 1 above)
Comment (MrAE) [reply R/fpPredict.R]: After thinking about this, a matrix is required to have entries of one type, so this is fine for a matrix.

File: R/fpRerF.R (hunk 2 above)
Comment (MrAE) [reply R/fpRerF.R]: After thinking about this, a matrix is required to have entries of one type, so this is fine for a matrix.

File: R/fpRerF.R [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+		if(is.null(numTreeBins)){
+			numTreeBins <- numCores
+		}
+
+		forest_module <- methods::new(forestPackingRConversion)
+		forest_module$setParameterString("forestType", forestType)
+		forest_module$setParameterInt("numTreesInForest", numTreesInForest)
+		forest_module$setParameterInt("minParent", minParent)
+		forest_module$setParameterInt("numCores", numCores)
+		forest_module$setParameterInt("useRowMajor",0)
```
[the remark this answers is not in the document]
Comment (MrAE) [reply R/fpRerF.R]: @falkben as opposed to vector input? My first thought is that it wouldn't matter. a column vector would be stored the same as a row vector (I think?)

[the remark this answers is not in the document]
Comment (falkben) [reply R/fpRerF.R]: as opposed to the CSV input, in which we can index by row/column directly.  this setting is I think irrelevant I think if loading data through CSV so I'm not sure we should be setting it. perhaps @jbrowne6  can comment. it shouldn't hurt anything, but we also probably shouldn't be setting parameters if they aren't needed, as it potentially makes things more confusing.

**Comment by falkben** [comment]
i'd like some simple tests to ensure these bindings are working as intended.  something similar to these tests: 

https://github.com/neurodata/RerF/blob/pybind/Python/tests/test_RerF.py
https://github.com/neurodata/RerF/blob/pybind/Python/tests/test_packedForest.py

**Comment by falkben** [comment]
closed in favor of https://github.com/neurodata/RerF/pull/182

**Pull request closed without merging**